packages/react-dom/src/events/ReactDOMEventReplaying.js

```diff
@@ -540,15 +557,20 @@ function attemptExplicitHydrationTarget(
 
 export function queueExplicitHydrationTarget(target: Node): void {
   if (enableSelectiveHydration) {
-    const priority = getCurrentPriorityLevel();
+    const schedulerPriority = getCurrentPriorityLevel();
+    const updateLanePriority = higherLanePriority(
```

Comment:
What's this part for?

Reply:
The scheduler priority defaults to `NormalPriority` but the `currentUpdateLanePriority` defaults to `NoLanePriority` (so that we can tell whether we've set the priority or not).

We can remove this for now since we're not using the lane priority in this function anymore, but when it was changing the queue order we needed to use the default priority to match the same order as the scheduler.

Reply:
I would check for `NoLanePriority` explicitly